<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessControlDialog</class>
 <widget class="QDialog" name="AccessControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyPassX/Http: Confirm Access</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="itemsList"/>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberDecisionCheckBox">
     <property name="text">
      <string>Remember this decision</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="allowButton">
       <property name="text">
        <string>Allow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="denyButton">
       <property name="text">
        <string>Deny</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
